test/dummy/config/initializers/google_sign_in.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.configure do
   config.google_sign_in.client_id = FAKE_GOOGLE_CLIENT_ID
   config.google_sign_in.client_secret = FAKE_GOOGLE_CLIENT_SECRET
+
+  # Default changed to basic auth. Use old :request_body for the sake of our test stubs.
+  config.google_sign_in.oauth2_client_options = { auth_scheme: :request_body }
```

Comment:
Breaking change in oauth2 2.0. Auth scheme changed default value from request body to basic auth. Stick with the old scheme for tests so the existing stubs work.